pkg/kubectl/bind-apiservice/plugin/servicebindings.go

```diff
@@ -139,13 +141,182 @@ func (b *BindAPIServiceOptions) createAPIServiceBindings(ctx context.Context, co
 	return bindings, nil
 }
 
+//Print Permission Claim to provided Writer
```

Comment:
**suggestion** This comment provides little additional information over the function signature. Could you expand or remove it?